Makefile

```diff
@@ -9,17 +8,20 @@ test_files =$(wildcard tests/**/*.ts)
 test_out_files = $(test_files:tests/%.ts=tests/%.lua)
 test_out_init = $(test_init:tests/%.ts=tests/%.lua)
 
-compile:
+install:
+	npm ci
+
+compile: install
```

Comment:
**suggestion**: Keep `compile` as the first target because it gets executed by default when calling `make` without parameters